Resources/Prototypes/DeltaV/Entities/Structures/Doors/Windoors/windoor.yml

```diff
@@ -69,3 +69,96 @@
   - type: ContainerFill
     containers:
       board: [ DoorElectronicsLawyer ]
+
+- type: entity
+  parent: WindoorSecure
+  id: WindoorSecureMantisLocked
+  suffix: Mantis, Locked
+  components:
+  - type: ContainerFill
+    containers:
+      board: [ DoorElectronicsMantis ]
+
+# Delta V: Add Airlocks/upstream roles
```

Comment:
its a deltav file :trollface:

Reply:
> its a deltav file :trollface:

It's in the other DV file the same 😎. I guess just indicating these are not "our" roles 🤷‍♂️